code/modules/mob/mob.dm

```diff
@@ -1969,13 +1965,13 @@ mob/proc/on_foot()
 /mob/proc/AdjustPlasma()
 	return
 
-/mob/living/carbon/heard(var/mob/living/carbon/human/M)
+/mob/heard(var/mob/M)
```

Comment:
suspicion that your heard list is going to get filled up with stuff like bees, chickens, Ian, etc.